src/components/CustomHooks/useCity.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+
+import stateCity from "../../common/utils/state-city.json";
+
+const getCity = (state) => {
+  const [cities, setCities] = useState([]);
```

Comment:
Better to start custom hook name with 'use' as well for more clarity. I can see it's in CustomHooks folder but still.

Reply:
done